### PR TITLE
List buckets re-design

### PR DESCRIPTION
**Pull request by dvaldivia**
New Reactive design.

**To Test this change you must use MinIO Master**

Screen Shot 2021-09-17 at 6 55 41 PM

### CODE REVIEW

**Review by adfost — DISMISSED** [review]

**Review by adfost — DISMISSED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 1]
```diff
@@ -268,55 +274,114 @@ const ListBuckets = ({
...
       )}
-      <PageHeader label={"Buckets"} />
+      <PageHeader
+        label={"Buckets"}
+        actions={
+          <Fragment>
+            <Grid
+              container
+              direction="row-reverse"
+              justifyContent="flex-end"
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:284]: This is not a valid property for mui Grid

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 2]
```diff
@@ -268,55 +274,114 @@ const ListBuckets = ({
...
+              )}
+
+              <Grid item>
+                <Box display={{ xs: "block", sm: "block", md: "none" }}>
+                  <TextField
+                    className={classes.theaderSearch}
+                    variant={"outlined"}
+                    id="search-resource"
+                    placeholder={"Search Buckets"}
+                    labelWidth={0}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:309]: This is not a valid property for this component

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please review TextField component usage under ListBuckets

**Review by bexsoft — APPROVED** [review]
LGTM, Need to delete node_modules & make `yarn install` again to run this.

**Review by Alevsk — APPROVED** [review]

**Pull request merged**